src/main/java/io/confluent/kafkarest/converters/AvroConverter.java

```diff
@@ -67,10 +68,16 @@
   public static Object toAvro(JsonNode value, Schema schema) {
     try {
       ByteArrayOutputStream out = new ByteArrayOutputStream();
+      value = JsonNodeConverters.transformJsonNode(value, schema, new JsonNodeConverter() {
```

Comment:
i dont recall exactly how the JVM handles this (maybe it does optimally), but it might be more efficient to have your implementation of JsonNodeConverter defined somewhere rather than repeatedly created